### DISCUSSION TITLE
ERROR: ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (lxml)

### DISCUSSION THREAD

**Discussion by TMScience** [nianeyna/ao3downloader#148] [Q&A]
As the title says, this is the error I am getting even after un-installing and re-installing. Not too sure where to go from here and would love any help/advice (: thank you!

```ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (lxml)```

**Comment by nianeyna** [comment]
❗ Checkpoint! ❗

Have you done ALL of the troubleshooting steps in the README? (The README is the explanatory text on the landing page for ao3downloader. You can find the Troubleshooting section using the Table of Contents at the beginning of the README.)

If you have done all the troubleshooting steps, please tell me what the results were. If you haven't, please do them to the best of your ability, and tell me what the results were. ✨ 

❗ Thank you! ❗
